packages/hub/src/lib/list-datasets.ts

```diff
@@ -69,6 +73,7 @@ export async function* listDatasets<
 		...Object.entries({
 			limit: String(Math.min(totalToFetch, 500)),
 			...(params?.search?.owner ? { author: params.search.owner } : undefined),
+			...(params?.search?.query ? { query: params.search.query } : undefined),
```

Comment:
```suggestion
			...(params?.search?.query ? { search: params.search.query } : undefined),
```
this is search here no?